static/bidder-params/conversant.json

```diff
@@ -24,10 +24,6 @@
       "type": "integer",
       "description": "Ad position on screen."
    	},
-   	"mobile": {
```

Comment:
The docs for adapters are now located in the prebit.github.io repo. The link for this adapter is:
https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/conversant.md

The `mobile` param isn't listed there, so I don't think any updates are needed from this PR.